chart/apisix/values.yaml

```diff
@@ -95,34 +102,51 @@ admin:
     admin: edd1c9f034335f136f87ad84b625c8f1
     viewer: 4054f7cf07e344346cd3f287985e76a2
 
+allow:
+  # The ip range for allowing access to Apache APISIX
+  ipList:
+    - 127.0.0.1/24
+
 # APISIX plugins to be enabled
 plugins:
-  - example-plugin
-  - limit-req
-  - limit-count
-  - limit-conn
-  - key-auth
+  - api-breaker
+  - authz-keycloak
   - basic-auth
-  - prometheus
-  - node-status
-  - jwt-auth
-  - zipkin
-  - ip-restriction
+  - batch-requests
+  - consumer-restriction
+  - cors
+  - echo
+  - fault-injection
   - grpc-transcode
-  - serverless-pre-function
-  - serverless-post-function
+  - hmac-auth
+  - http-logger
+  - ip-restriction
+  - jwt-auth
+  - kafka-logger
+  - key-auth
+  - limit-conn
+  - limit-count
+  - limit-req
+  - node-status
   - openid-connect
+  - prometheus
+  - proxy-cache
+  - proxy-mirror
   - proxy-rewrite
   - redirect
+  - referer-restriction
+  - request-id
+  - request-validation
   - response-rewrite
-  - fault-injection
+  - serverless-post-function
+  - serverless-pre-function
+  - sls-logger
+  - syslog
+  - tcp-logger
   - udp-logger
+  - uri-blocker
   - wolf-rbac
-  - proxy-cache
-  - tcp-logger
-  - proxy-mirror
-  - kafka-logger
-  - cors
+  - zipkin
 stream_plugins:
```

Comment:
ditto

Reply:
I keep it here is to leave a configuration item(placeholder) for users who wanted to use plugins belong themself.
In fact, we can indeed set the plug-in list to be empty, I made it compatible.
But I am not sure if we should set the plugin list empty by default?